db/migrate/20210324043327_delete_orphan_post_revisions.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class DeleteOrphanPostRevisions < ActiveRecord::Migration[6.0]
```

Comment:
Should this be a migration or a post_migration? 🤔

Reply:
The app doesn't depend on this change and it's potentially a slow SQL, so I'd move it into a post migration.